packages/mobile/src/firebase/db-rules.json

```diff
@@ -31,6 +31,9 @@
         "appVersion": {
           ".write": "auth.uid != null && $address === root.child('users').child(auth.uid).child('address').val()"
         },
+        "country": {
+          ".write": "auth.uid != null && $address === root.child('users').child(auth.uid).child('address').val()"
```

Comment:
Hmm, shouldn't this be `false` if the cloud function is the one updating it?

Reply:
🥲 i initially thought `false` would prevent everyone from updating it including the cloud function...but this clearly isn't the case, i see `tokens` for example also has `false` so i've updated this now!

Reply:
Someone with an admin key (like our cloud functions) can always read/write, the permissions are for end users :)